docs/frontend/ajax-client.md

```diff
@@ -125,3 +128,13 @@ In your controller you will need to check if the CSRF token is valid:
     throw new InvalidCsrfTokenException('Invalid CSRF token');
   }
 ```
+
+## Busy button spinner
+
+The content of a clicked button can be replaced by a spinner during the request. Pass the DOM Node(s) like below:
+
+```javascript
+ajaxClient.busy_targets = [buttonNode]
```

Comment:
Gaat dat niet altijd vergeten worden? Kunnen we niet zorgen dat dat volledig automatisch is?